packages/odyssey-react/src/components/Title/index.tsx

```diff
@@ -36,7 +37,7 @@ export interface Props
   /**
    * The human readable section title to be visually displayed
    */
-  children: ReactText;
+  children: ReactText | ReactNode;
```

Comment:
`ReactNode` will allow any valid JSX child, which means the union operator with `ReactText` is redundant at this point. What is the intention with this change?

Reply:
It resolved [an error](https://github.com/okta/odyssey/pull/1076/files#diff-2f4b0e8d64e2c744e0cb9a0cd9c02806753359a7cfaea45563055356ed5e37beR508-R510) that popped up in an example story!